apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,182 @@
+
+### Create a new Project and Authoring Environment 
+
+1. From there, create a new project with the XM Cloud Deploy App by clicking `Create project`.  
+![Project Overview - Create new Project](https://sitecorecontenthub.stylelabs.cloud/api/public/content/2215cae974524bcb820ce28fd7fe3c51?v=df12ee70)  
+2. From here you provide a Project Name e.g.”XM Cloud Tutorial Series” and click the Continue button 
+![Create Project and Environment Step 1 - Provide Project Name](https://sitecorecontenthub.stylelabs.cloud/api/public/content/95a7fa3d5e7e4f6997494cae9a5ac70e?v=3678a3bd) 
+3. Choose whether you want Connect to GitHub or to Azure DevOps. A starter solution will be copied to your connected source code repository as a starting point. For the sake of this tutorial you choose GitHub and press the Continue Button   
+![Create Project and Environment Step 2 - Choose Source Code Repository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/27f501fc072d4a23b2a89a64b6e936f5?v=eafd675a) 
+
+In step 3 you can choose whether you want to start from the XM Cloud template or if you want to use your own code. The XM Cloud template is a starter kit based on Next.js, that contains already a rich set of features and sample implementations. It is best practice to start from the template.   
+
+4. Choose a GitHub Account from the dropdown or Connect to a new account by following the steps. Make sure you give rights to create a new repository in your account. 
+![Create Project and Environment Step 3 - Setup Respository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/3adbe7059311434d8b7900d691990e22?v=08e2526a) 
+5. Provide a name for the repository e.g.xm-cloud-tutorial-series and click the Continue button. 
+6. Enter a name for the environment e.g. DEV  
+![Create Project and Environment Step 4 - Provide Environment details](https://sitecorecontenthub.stylelabs.cloud/api/public/content/7f69f87c697148afabc6e55526588d4f?v=751f0044) 
+7. Select whether this is a production environment or not. You select “No” 
+8. Select whether you want to autodeploy on push to the repository. Select “Yes”. This enables the CI/CD pipeline from your main branch. This can be adjusted later. 
+9. Press the Continue button. 
+10. On the 5th step review your selections and press the “Start deployment” button
+![Create Project and Environment Step 5 - Review your selections](https://sitecorecontenthub.stylelabs.cloud/api/public/content/d5cda1bf224c4f99b508fe612e527590?v=6c0f0076) 
+11. The deployment starts, and provisioning and build run in parallel.  
+![XM Cloud Deploy Logs - Deployment is running](https://sitecorecontenthub.stylelabs.cloud/api/public/content/81ff5cf671f24d948aeb0dfb6d769411?v=4abce4be) 
+**A note on provisioning and build:**  
+*The Provisioning sets up all the resources you need to run the XM Cloud instance, while the Build is building the software solution you have cloned to your personal repository. This will take a few minutes.*  
```

Comment:
I'd add: 'Read on for more information on what is going on ‘under the hood’ as your project sets up. '  after '...this will take a few minutes'

Reply:
This seems to be resolved